### PR TITLE
fixed _low_precision to throw the warning when off the table

### PR DESCRIPTION
**Pull request by kvyh**
This should fix the error in the `test_iers_download`.

### CODE REVIEW

**Comment by bmorris3** [comment]
@kvyh Looks like that didn't quite do it.

File: astroplan/utils.py [hunk 1]
```diff
@@ -37,8 +37,12 @@ def _low_precision_utc_to_ut1(self, jd1, jd2):
...
     """
     try:
-        return self.delta_ut1_utc
-    except IndexError:
+        if self.mjd*u.day not in iers.IERS_Auto.open()['MJD']:
+            warnings.warn(IERS_A_WARNING, OldEarthOrientationDataWarning)
+            return self.delta_ut1_utc
+        else:
+            return self.delta_ut1_utc
+    except ValueError:
```
Comment (bsipocz) [inline astroplan/utils.py:45]: Why did you change the error type?

 Is there any scenario when it goes into the exception part, assuming that now astroplan requires astropy >=1.2?

**Comment by kvyh** [comment]
@bsipocz would it be possible to restart the 24.13 check to see if it is working for python2.7?

File: astroplan/utils.py (hunk 1 above)
Comment (kvyh) [inline astroplan/utils.py:45]: I changed the error type because the IndexError was never occuring (which is what got fixed by astropy 1.2), but ValueErrors were caused  a lot when it tried to evaluate the iers table.

Comment (bsipocz) [inline astroplan/utils.py:45]: OK, great. Merging then.
